### ISSUE TITLE
When setting device to Silent mode, TalkBack must still speak!

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#381]
```
What steps will reproduce the problem?
1. On a Nexus 7 (and probably other devices), turn on TalkBack.
2. Long-press the on/off button until the Shut Down menu appears.
3. Explore or swipe to Silent Mode, and double-tap.

What is the expected output? What do you see instead? Expected: Device should 
go to Silent Mode, but TalkBack should still speak. Instead, the device goes to 
Silent mode, but TalkBack falls silent, too, leaving the blind user with a 
device that they can no longer operate.


What version of the product are you using? On what operating system? Both 
TalkBack 3.5.1 as well as 3.5.2Beta reproduce this on a Nexus 7 with Android 
4.4.4.


Please provide any additional information below.

Nothing else to report, other than silencing TalkBack should be a different 
scenario than silencing the device, and it would need a bullet-proof way to 
recover from. I was only able to "unbrick" my device by performing the above 
steps again from muscle memory and, by counting swipes, hit the correct option 
to turn Silent mode back off.
```

Original issue reported on code.google.com by `marco.z...@gmail.com` on 17 Aug 2014 at 6:02
